### ISSUE TITLE
Cannot Trigger Magic Ritual

### ISSUE THREAD

**Issue by Jackiecrazy** [Polarice3/Goety-2#8]
![2023-04-17_15 06 27](https://user-images.githubusercontent.com/6895485/232410496-ea5f92e5-f5af-43f9-92c9-916773aea40c.png)
I have far more than enough blocks for the occasion. Is it perhaps because Apotheosis replaces the enchantment table? Even so, their implementation of the replacement still extends EnchantmentTableBlock...
On the same note, a message telling us what we're missing would be very useful for these occasions.

**Comment by Polarice3** [comment]
Does one of the pedestal contain a Hunger Core? It appears you're trying to create a Feasting Focus.

**Comment by Jackiecrazy** [comment]
It does indeed. The ender pearl and the hunger core are hidden by the bookshelf and the altar respectively. I also tried taking them out, and it seems like you won't actually get the "Inefficient Requirements!" message unless you have all the required item components?

**Comment by Polarice3** [comment]
I'm testing it with Apotheosis right now.

**Comment by Polarice3** [comment]
Alright, ritual doesn't work with Apotheosis. I think you're correct in that it might be the Enchanting Table, but I'm not entirely sure. I'm gonna need to check and test this out.  In the meantime, I'll check out the Forge Tag for Bookshelves.

**Comment by Jackiecrazy** [comment]
Just checked again, and it still doesn't work. On a similarly bizarre note, I just managed to perform an animation ritual without having a carved pumpkin. Something is most likely wrong with my mod list, but it is also nightmarishly long.
![2023-04-17_17 33 16](https://user-images.githubusercontent.com/6895485/232445304-29a3cc6e-0335-4c55-a1a9-fc4d0fea1bab.png)

**Comment by Polarice3** [comment]
I'll look into the Ritual block requirements closer to make sure everything works properly. Sorry for the inconvenience.

**Comment by Jackiecrazy** [comment]
Oh! That could explain it. I put the Black Book on it. Let me try again.

**Comment by Jackiecrazy** [comment]
aaaaaand now it works perfectly! I probably just left a carved pumpkin somewhere around my house to prank myself randomly at a future date. It certainly worked this time.

**Comment by Polarice3** [comment]
Alright, next update I'll make the ritual work if Lectern has any book on them! Thank you! :)
